articles/tokens/overview-access-tokens.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Access Tokens
+description: Learn what Access Tokens are and how you can use them with Auth0.
+toc: true
+topics:
+  - tokens
+  - access-tokens
+contentType:
+  - concept
+useCase:
+  - invoke-api
+---
+# Access Tokens
+
+An Access Token is a credential that can be used by an application to access an API. Access Tokens can be either an opaque string or a JSON web token. They inform the API that the bearer of the token has been authorized to access the API and perform specific actions specified by the **scope** that has been granted. 
+
+Access Tokens should be used as a **Bearer** credential and transmitted in an HTTP **Authorization** header to the API. 
+
+Depending on how your application needs to use the Access Token, you can do the following:
+
```

Comment:
Looks like all the links (except for rules) are broken

Reply:
Fixed